install.lua

```diff
@@ -0,0 +1,64 @@
+-- Easy Installer for computercraft-github by Eric Wieser
+-- https://github.com/eric-wieser/computercraft-github
+
+local tree = select(1,...)
+if not tree then
+  tree = 'master'
+end
+
+local REPO_BASE = ('https://raw.githubusercontent.com/eric-wieser/computercraft-github/%s/'):format(tree)
+
+local FILES = {
+  'apis/dkjson',
+  'apis/github',
+  'programs/github',
+  'github'
+}
+
+local function request(url_path)
+  local request = http.get(REPO_BASE..url_path)
+  status = request.getResponseCode()
+  response = request.readAll()
+  request.close()
+  return status, response
+end
+
+local function makeFile(file_path, data)
+ local file = fs.open('github.rom/'..file_path,'w')
+ file.write(data)
+ file.close()
+end
+
+local function rewriteDofile(filename, required)
```

Comment:
Nice! This should probably look for quotes and/or `dofile` around the string though